hwy/detect_compiler_arch.h

```diff
@@ -303,10 +303,29 @@
 #define HWY_ARCH_S390X 0
 #endif
 
+#if defined(__loongarch64__) || \
```

Comment:
I can't find the definition of this macro; should it be `__loongarch64`?

Reply:
Ah, I see some code with __loongarch64__ and more with __loongarch64. I suppose we could do both? Or are we sure only the latter is correct?

Reply:
As far as I know, `__loongarch64__` was an earlier internal usage, which was later adjusted to `__loongarch64`. Considering compatibility, both can be kept. Thanks.

Reply:
Got it, thanks!